server/src/main/java/org/opensearch/action/admin/cluster/remotestore/stats/RemoteStoreStats.java

```diff
@@ -151,6 +208,38 @@ static final class Fields {
         static final String REMOTE_REFRESH_LATENCY_IN_MILLIS = "remote_refresh_latency_in_millis";
     }
 
+    static final class DownloadStatsFields {
+        /**
+         * Last successful segment download timestamp in milliseconds
+         */
+        static final String LAST_DOWNLOAD_TIMESTAMP = "last_download_timestamp";
+
+        /**
+         * Total number of segment files downloaded from the remote store for a specific shard
+         */
+        static final String TOTAL_SYNCS_FROM_REMOTE = "total_syncs_from_remote";
```

Comment:
can we also rename the `static final String TOTAL_REMOTE_REFRESH = "total_remote_refresh";` to `static final String TOTAL_SYNCS_TO_REMOTE = "total_syncs_to_remote";` for the sake of consistency.

Reply:
Changed in line 184